test/EventStream-e2e-node.js

```diff
@@ -0,0 +1,29 @@
+/*
+
+End-to-end test program for the event stream with Node
+
+Steps:
+- npm run compile
+- PARTICLE_API_TOKEN=<my-token> node test/EventStream-e2e-node.js
+- Follow the scenarios in EventStream.feature
+
+ */
+
+const Particle = require('../lib/Particle');
+
+const baseUrl = process.env.PARTICLE_API_BASE_URL || 'http://localhost:9090';
+const auth = process.env.PARTICLE_API_TOKEN;
+
+const particle = new Particle({ baseUrl });
+
+particle.getEventStream({ deviceId: 'mine', auth }).then(stream => {
+	console.log('event stream connected');
+
+	['event', 'error', 'disconnect', 'reconnect', 'reconnect-success', 'reconnect-error'].forEach(eventName => {
+		stream.on(eventName, (arg) => {
+			console.log(eventName, arg);
+		});
+	});
+}).catch(function (err) {
+	console.error(err);
+});
```

Comment:
what's the reason for having both `.html` and `.js` versions of this file?

Reply:
One is for Node and one is for the browser.